.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea
 *.tgz
+*.swp
```

Comment:
Stuff like this can be put into `~/.gitignore` so that you don't need to copy it into every single repo.